api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -282,6 +282,14 @@ func (in *IoChaosSpec) DeepCopyInto(out *IoChaosSpec) {
 		*out = make([]string, len(*in))
 		copy(*out, *in)
 	}
+	if in.NextStart != nil {
+		in, out := &in.NextStart, &out.NextStart
+		*out = (*in).DeepCopy()
+	}
+	if in.NextRecover != nil {
+		in, out := &in.NextRecover, &out.NextRecover
+		*out = (*in).DeepCopy()
+	}
```

Comment:
Maybe you need to update the branch. We made some updates . 
the `nextStart` and `nextRecover` have moved to `status`

#514

Reply:
Done.